yt/loaders.py

```diff
@@ -1460,52 +1463,14 @@ def flatten(l):
 
 
 # --- Loader for yt sample datasets ---
-def load_sample(
+@parallel_root_only_then_broadcast
+def _get_sample_data(
     fn: Optional[str] = None, *, progressbar: bool = True, timeout=None, **kwargs
```

Comment:
just realized that `kwargs` here probably needs special treatment... if this functions modifies it (which it **can** do), then that would only happen on the root process.